src/mixins/filing-template-mixin.ts

```diff
@@ -209,7 +213,9 @@ export default class FilingTemplateMixin extends DateMixin {
     this.setOfficeAddresses(draftFiling.incorporationApplication.offices)
 
     // restore Name Translations
-    this.setNameTranslationState(draftFiling.incorporationApplication.nameTranslations || [])
+    if (draftFiling.incorporationApplication.nameTranslations) {
+      this.setNameTranslations(draftFiling.incorporationApplication.nameTranslations)
+    }
```

Comment:
I think I prefer this style, where we don't overwrite the store initial values (and so we don't set the initial value a second time and in a second place).